util/logtools/encounter_tools.ts

```diff
@@ -445,15 +449,13 @@ class TLFuncs {
       const startTime = fight.startTime ? TLFuncs.timeFromDate(fight.startTime) : 'Unknown Start';
       const fightDuration = TLFuncs.durationFromDates(fight.startTime, fight.endTime) ?? 'Unknown Duration';
       let fightName = 'Unknown Encounter';
-      if (fight.sealName)
+      if (fight.sealName !== undefined && fight.sealName.length > 0)
         fightName = fight.sealName;
-      else if (fight.fightName)
+      else if (fight.fightName !== undefined && fight.fightName.length > 0)
```

Comment:
```suggestion
      else if (fight.fightName !== undefined)
```

`fightName` comes from the zone or seal, so shouldn't be empty.